canvas_modules/common-canvas/src/common-properties/controls/structuretable/structuretable.scss

```diff
@@ -16,5 +16,4 @@
 
 .properties-st.disabled {
 	pointer-events: none;
-	opacity: 0.5;
```

Comment:
Accessibility for text resolved:
![image](https://github.com/user-attachments/assets/cd01e527-d5f3-4aec-9047-80d444b37679)